agent/containers/images/verify-tags

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [[ -z "${1}" ]]; then
+    printf -- "No tags files given to process!\n" >&2
+    exit 1
+fi
+
+# Use the first argument as the check-sum to verify against.
+cksum="$(cksum ${1} | awk '{print $1}')"
+shift
+
+ret_val=0
+while [[ ! -z "${1}" ]]; do
+    new_cksum="$(cksum ${1} | awk '{print $1}')"
+    shift
```

Comment:
@portante I am unable to understand how cksum in ln.9 is different from new_cksum in ln.14

Reply:
The `shift` built-in of `bash` removes the first command line argument and "shifts" the remaining so that the previously second argument is now the first, and so on.

Reply:
I had to read this twice to figure out the concern and the explanation. `verify-tags` takes a list of file arguments, calculates the checksum of the first file, and compares all subsequent file checksums against that value. Without searching for and analyzing the uses of this script it's not clear exactly why we're doing that or what it means, which I think was the point of @riya-17's comment.

Could we have some more clear documentation?

Reply:
This is a backport of existing code in `master` so if we need to make changes to this script, we'll have to do it there first.

As a matter of behavior, `verify-tags` (called from the `Makefile` @ [line 178](https://github.com/portante/pbench/blob/backport-f33-centos8-cont-fixes/agent/containers/images/Makefile#L178)) verifies that the generated `*-tags.lis` files are all identical.  If they are not identical, it means we have different versions of software built across the release targets.

Each COPR build target ("Fedora 33, Fedora 32, EPEL 8, EPEL 7") has a separate RPM repository in COPR.  Each `<target>-tags.lis` file is generated by inspecting that repository for that target (see `Makefile` @ [line 173](https://github.com/portante/pbench/blob/backport-f33-centos8-cont-fixes/agent/containers/images/Makefile#L173)).